### PR TITLE
sys-apps/yarn: 1.12.3 version bump

### PR DESCRIPTION
**Pull request by saboya**

### CODE REVIEW

File: sys-apps/yarn/yarn-1.12.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline sys-apps/yarn/yarn-1.12.3.ebuild]: You can bump this to 7.

File: sys-apps/yarn/yarn-1.12.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+src_install() {
+	local install_dir="/usr/$(get_libdir)/node_modules/yarn" path shebang
+	insinto "${install_dir}"
+	doins -r .
+	dosym "../$(get_libdir)/node_modules/yarn/bin/yarn.js" "/usr/bin/yarn"
+
+	while read -r -d '' path; do
+		read -r shebang < ${path} || die
+		[[ "${shebang}" == \#\!* ]] || continue
+		chmod +x "${path}" || die #614094
```
Comment (juippis) [inline sys-apps/yarn/yarn-1.12.3.ebuild]: Would fperms work here? No need to die when using it.

File: sys-apps/yarn/yarn-1.12.3.ebuild (hunk 1 above)
Comment (saboya) [reply sys-apps/yarn/yarn-1.12.3.ebuild]: Done.

File: sys-apps/yarn/yarn-1.12.3.ebuild (hunk 2 above)
Comment (saboya) [reply sys-apps/yarn/yarn-1.12.3.ebuild]: Changed it so it uses fperms.

**Comment by hydrapolic** [comment]
Seems like it has already been merged into Gentoo.

**Pull request closed without merging**